app/templates/main.html

```diff
@@ -1,7 +1,18 @@
 <section>
 	<div>
+		<form>
+			<label>Add a Room!<br>
+			<input type="text" name="add_room" ng-model="roomName" placeholder="Room Name"></label>
+			<input type="button" value="Add Room!" ng-click="main.addRoom(roomName)">
+		</form>
 		<ul ng-repeat="room in rooms">
 			<li>{{room.$value}}</li>
```

Comment:
Using `room.$value` is not what we want. It works here because of the problem noted in `Room.js` with `$add`. That problem means you are storing rooms with no key, just a value. So instead of:

```
{
  name: 'Room1'
}
```

...you are storing:

```
'Room1`
```

See https://github.com/firebase/angularfire/blob/master/docs/reference.md#value. Once `$add` is fixed this should be more like `{{room.name}}`